CS483/CS483/Kartaclysm/Data/UserConfig/ControlBindings.xml

```diff
@@ -1,54 +1,54 @@
 <ControlBindings>
   <Keyboard1>
-      <accelerate value="265" />
-      <brake value="264" />
-      <left value="263" />
-      <right value="262" />
-      <slide value="32" />
-      <driverAbility1 value="65" />
-      <driverAbility2 value="83" />
-      <kartAbility1 value="68" />
+      <accelerate value="87" />
+      <brake value="83" />
+      <left value="65" />
+      <right value="68" />
+      <slide value="90" />
+      <driverAbility1 value="81" />
+      <driverAbility2 value="69" />
+      <kartAbility1 value="82" />
       <kartAbility2 value="70" />
-      <pause value="257" />
+      <pause value="88" />
```

Comment:
Ooooh, I see. I'm not sure the defaults need to be changed though. The default control scheme is still what we want when we're not testing using four players on the keyboard. I just edited the user config to override the defaults for testing this way.